packages/suite/src/views/wallet/coinmarket/common/CoinmarketOffers/CoinmarketOffersItem.tsx

```diff
@@ -156,29 +144,32 @@ const CoinmarketOffersItem = ({ quote }: CoinmarketOffersItemProps) => {
                         </Row>
                     </OfferColumn2>
```

Comment:
Can we please call those columns better?

e.g. 
`ExchangeNameOfferColumn`, `AmountOfferColumn`, `ActionsOfferColumn`,..

Reply:
ok, done